day-26-challenge/solution.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.map = function (array, callback) {
```

Comment:
callback should support (element, index, array) to function the same as the array methods